dev-client/src/screens/LocationScreens/components/CreateSiteButton.tsx

```diff
@@ -25,15 +25,15 @@ import {Coords} from 'terraso-client-shared/types';
 import {Icon} from 'terraso-mobile-client/components/icons/Icon';
 import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
 
-type Props = {coords: Coords};
+type Props = {coords: Coords; elevation?: number};
```

Comment:
Question: Why is `elevation` optional -- is there ever a case where we wouldn't have a value for `elevation` here?

Reply:
It's possible the elevation API could fail.